.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 Thanks for your hard work, please ensure all items are complete before opening a PR.
 
-- [ ] You have signed the CLA as per the instructions in [CONTRIBUTING.md](https://github.com/cloudant/couchbackup/blob/master/CONTRIBUTING.md#contributor-license-agreement)
+- [ ] Tick to sign-off your agreement to the [DCO 1.1](https://github.com/cloudant/couchbackup/blob/master/DCO1.1.txt)
```

Comment:
Can this link be made relative? We use a relative link to it in CONTRIBUTING. There are pros and cons (depending on whether the user is viewing on github or in a checked out repo etc)